module/dice/damageRoll.mjs

```diff
@@ -137,9 +137,15 @@ export default class DamageRoll extends DHRoll {
         }
 
         if (config.isCritical && part.applyTo === CONFIG.DH.GENERAL.healingTypes.hitPoints.id) {
-            const tmpRoll = Roll.fromTerms(part.roll.terms)._evaluateSync({ maximize: true }),
-                criticalBonus = tmpRoll.total - this.constructor.calculateTotalModifiers(tmpRoll);
-            part.roll.terms.push(...this.formatModifier(criticalBonus));
+            let total = 0;
+            part.roll.terms.forEach(term => {
```

Comment:
This works fine in function. I think we could do it a little more neatly though:
```
let total = part.roll.dice.reduce((acc, term) => acc + term._faces*term._number, 0);
```

Checking dice instead of terms lets us skip verifying that it's a dice term, and reduce is tailor made to increase a number while looping through.